src/components/logTable/logTable.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { List, Item, Form, Message, Button } from '../../../node_modules/semantic-ui-react';
+import { List, Form, Button } from '../../../node_modules/semantic-ui-react';
```

Comment:
since semantic ui react is a node module, I guess you should be able to import directly from 'semantic-ui-react'?

Reply:
I think you are right, I didn't have to for any of my components and it worked fine. This is for Larry's log table, I only edited it because the unused imports made me fail some tests. I or Larry can go back and fix it, or we can fix it later, since some of the code, notably the ref stuff, is going to need to cleaned up in the future anyway. Up to y'all.

Reply:
Okay, then we'll clean this up together later. I'll approve this change for now.